libs/contracts/README.md

```diff
@@ -1,3 +1,19 @@
-# FeatureBoard API Contracts
+# FeatureBoard Contracts
 
-Contains TypeScript type definitions for the WebSocket API message contracts of https://featureboard.app
+Contains TypeScript type definitions for FeatureBoard SDK
+
+## Installation
+
+The FeatureBoard Contracts is available on NPM.
+
+```bash
+npm add @featureboard/contracts
+# Or
+yarn add @featureboard/contracts
+# Or
+pnpm add @featureboard/contracts
+```
+
+## Release notes
+
+You can find our [changelog here](CHANGELOG.md).
```

Comment:
Does this get published with the NPM package? This readme will show on npmjs.org so links need to work after it gets published. Might need to be a fully qualified link

Reply:
I did some research about it and what I understand is that it should work. I will double check, otherwise I just link to github like I have for the dotnet project...

Reply:
It will link to github when published to npm as it is now.